GovUk.Frontend.AspNetCore.Extensions/Views/Shared/GOVUK/Pagination.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@using System.Globalization;
+@using System.Web;
+@addTagHelper *, GovUk.Frontend.AspNetCore
+@model GovUk.Frontend.AspNetCore.Extensions.Models.PaginationModel
+@if (Model != null && Model.PageSize.HasValue && Model.TotalItems > Model.PageSize)
+{
+    var path = Model.Path ?? Context.Request.Path;
+    var queryString = Model.QueryString ?? Context.Request.QueryString;
+    var query = HttpUtility.ParseQueryString(queryString.ToString());
+    query.Remove(Model.QueryStringParameter);
+
+    var totalPages = (int)Math.Ceiling((decimal)Model.TotalItems / Model.PageSize.Value);
```

Comment:
There is a method on the `PaginationModel` called `TotalPages()`. Any reason why we can't use that rather than duplicate the calculation?

Reply:
Good point and this is something I already spotted for #138, which will replace the version you're looking at in the next sprint.  This branch is needed for now but will soon be obsolete.